<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VISA QT</string>
  </property>
  <widget class="QGroupBox" name="Status_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>STATUS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label_65">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid transparent;
color: red;</string>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-cursorPosition: 0;
border: 1px solid transparent;</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_connect_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-cursorPosition: 0;
border: 1px solid transparent;
background-color: rgba(255, 0, 0, 0.70)</string>
    </property>
    <property name="text">
     <string>Not Connected</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_66">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid transparent;
color: red;</string>
    </property>
    <property name="text">
     <string>Identification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	background-color: rgba(245, 188, 66, 150);
	border-radius: 5px;
	border: 1px solid transparent;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/usb.png</normaloff>../img/usb.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>0</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LG Smart UI SemiBold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  Instrument Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_67">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid transparent;
color: black;</string>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pText_cmd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="plainText">
    <string>W: Write Command
R: Read Command
T: Time Delay (ex. T: 0.2)
Option: Display Option</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color: rgba(66, 245, 102, 150);
	border-radius: 5px;
	border: 1px solid transparent;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightgreen;
}</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/send.png</normaloff>../img/send.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pText_monitor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>Return Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_68">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid transparent;
color: black;</string>
   </property>
   <property name="text">
    <string>Monitor</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
